modules/ListView.js

```diff
@@ -22,7 +21,44 @@
    */
   var ListView = View.extend((function() {
 
-    var removeChildView, addChildView;
+    var removeChildView, addChildView, aggregateRenders, breakDelayedRender;
```

Comment:
Why the additional functional scoping?  Aren't multiple ListViews going to get the same context anyway?  It would be clearer what is 'super private' if it lived outside the View.extend()